webots_ros2_tests/worlds/driver_test.wbt

```diff
@@ -0,0 +1,62 @@
+#VRML_SIM R2021b utf8
+WorldInfo {
+}
+Viewpoint {
+  orientation 0.15884326710103885 -0.5935655099023555 -0.7889542457906066 2.72544233076922
+  position -0.8954397740487239 1.2372617694112547 0.8102365388788932
+}
+TexturedBackground {
+}
+TexturedBackgroundLight {
+}
+Floor {
+  rotation 1 0 0 1.5708003061004252
+}
+CardboardBox {
+  translation 1.21847 0.00917563 0.3
+  rotation 0 0 1 1.309
+}
+Pioneer3at {
+  controller "<extern>"
+  extensionSlot [
+    LightSensor {
+      translation 0 0 0.29
+    }
+    Kinect {
+      translation 0.109998 9e-07 0.47
```

Comment:
Very minor comment: y-translation of Kinect, Solid and VelodyneVLP-16 can probably simplified to `0.0`.